src/test/java/com/nerodesk/om/SmallDocTest.java

```diff
@@ -87,7 +87,7 @@ public void failsToWriteBigDoc() throws IOException {
         );
         final String content = StringUtils.repeat('b', 101);
         final byte[] bytes = content.getBytes(StandardCharsets.UTF_8);
-        new SmallDoc(new MkDoc(file, "", ""), Tv.HUNDRED).write(
+        new SafeDoc(new SmallDoc(new MkDoc(file, "", ""), Tv.HUNDRED)).write(
```

Comment:
@dmzaytsev why do you need to decor here? The test should fail because of big file, not a zero size